website/content/en/docs/best-practices/multi-tenancy.md

```diff
@@ -0,0 +1,185 @@
+via the [subscription][olm-subscriptions]). Also, be aware that if the project was generated by Kubebuilder or SDK scaffold 
+then some values for the `Operator/manager`(See `config/manager/manager.yaml`) 
+are populated by default to get you started, however, you ought to optimize them based on your own tests and the specific needs of your operator.
+- **SHOULD** provide the mechanisms for [Monitoring compute & memory resource usage][k8s-metrics]. Kubebuilder and SDK provide some [metrics][kb-metrics] 
+which can help you with. Note that cluster admins can use it to monitor and resize the Operator and its Operands.
+**NOTE**  If the Operator is integrated with OLM and the bundle has a `PodMonitor` or a `ServiceMonitor` the 
+complete `InstallPlan` will fail on a cluster, which does not have these CRD/the Prometheus operator installed. 
+In this case, you might want to ensure the dependency requirement with [OLM dependency][olm-dep] or make clear 
+its requirement for the Operator consumers. 
+- **SHOULD** allow admins to customize the `requests/limits` amounts defined for the `Pod/Deployment`
+created by the Operator and not keep them fixed on the code.
+- **SHOULD** document how your Operator consumer can customize/rightsize the resources requests and limits for the 
+Operator and Operands `Pod/Deployments` or let its consumers know that the solution could adopt a better approach where 
+the Operator automatically adjust the values to the environment rather than asking its consumers to amend them
+
+You might also consider leveraging the [Vertical Pod Autoscaler][k8s-vertical-pod-scaling] to have the resources requested by the Operator 
+automatically adjusted to the cluster where it is deployed. You might also 
+look at allowing [horizontal pod autoscaling][k8s-autoscaling] for the other `Pod/Deployments` created by your project. 
+In this case, be aware that resource requests are also required for horizontal pod autoscaling to work.
+
+**NOTE:** If you are using OpenShift as Kubernetes distribution you might want to check the 
+doc [Automatically adjust pod resource levels with the vertical pod autoscaler
+][ocp-pod-scaling]. Caveat: If the VPA CRD is not available in the cluster where the operator 
+gets deployed the InstallPlan will fail. Please, see how to work with [OLM dependency][olm-dep] 
+if your project integrating with.
+
+### But, what happens
+
+#### when the resource requests are not set?
+
+**If not set:** 
+- configurations made by the cluster administrators such as [ResourceQuota][k8s-resources-quotas] might not work with the usage of the LimitRanger admission controller to force defaults might not work.
+- the Operators consumers might face resource shortages on a node when resource usage increases, for example, during a daily peak in request rate.
+- the Operators consumers might will be unable to install the Operator because does not have the minimal resources available.
```

Comment:
It is not fully clear to me. If resources are not requested by the operator deployment and it is scheduled on a node, which is already heavily utilized it may get starved or resources and as such times out (CPU) or fails (memory)  by startup. Would it cause an installation failure? I would tend to think that the installation is successful but the operator is not usable.

As I am not sure to follow your intent there is also another scenario:
if the operator DOES have resources requested and the cluster does not have capacity to accommodate it the pod won't be scheduled and stay in a pending state

Reply:
Change for `- the Operators consumers might be unable to successfully deploy the Operator because it does not have the minimal resources available.` we ought to not describe the details here in order to reduce the complexities